apps/client/src/routes/dashboards/dashboards-index/components/getting-started.tsx

```diff
@@ -0,0 +1,160 @@
+import { memo } from 'react';
+import {
+  Box,
+  Button,
+  ColumnLayout,
+  ContentLayout,
+  ExpandableSection,
+  Link,
+  SpaceBetween,
+} from '@cloudscape-design/components';
+import { FormattedMessage, useIntl } from 'react-intl';
+
+import { useApplication } from '~/hooks/application/use-application';
+import { CREATE_DASHBOARD_HREF } from '~/constants';
+import { gettingStartedColumnsTypes } from './types';
+import Dashboard from '../assets/Dashboard.svg';
+import Widget from '../assets/Widget.svg';
+import AssetsSearch from '../assets/AssetsSearch.svg';
+import './styles.css';
+
+const ValueWithLabel = ({
+  label,
+  children,
+}: {
+  label: string;
+  children: string;
+}) => (
+  <Box margin={{ bottom: 'xxl' }}>
+    <Box variant="awsui-key-label">{label}</Box>
+    <Box>{children}</Box>
+  </Box>
+);
+
+const GettingStarted = () => {
+  const intl = useIntl();
+  const { navigate } = useApplication();
+
+  const gettingStartedColumnArray: gettingStartedColumnsTypes[] = [
+    {
+      columnTitle: intl.formatMessage({
+        defaultMessage: 'Step 1: Create your dashboard',
+        description: 'step 1 title',
+      }),
+      icon: Dashboard,
+      className: 'getting-started-step1-icon',
+      extraMargin: '21px',
+      columnDescription: intl.formatMessage({
+        defaultMessage:
+          "Data models explorer defines data elements required to align your business objectives with Amazon Fraud Detector's data models.",
+        description: 'step 1 description',
+      }),
+      buttonTitle: intl.formatMessage({
+        defaultMessage: 'Create dashboard',
+        description: 'Create dashboard button title',
+      }),
+      handleCallback: () => {
+        navigate(CREATE_DASHBOARD_HREF);
+      },
+    },
+    {
+      columnTitle: intl.formatMessage({
+        defaultMessage: 'Step 2: Add widget',
+        description: 'step 2 title',
+      }),
+      icon: Widget,
+      className: 'getting-started-step2-icon',
+      extraMargin: '0px',
+      columnDescription: intl.formatMessage({
+        defaultMessage:
+          "Data models explorer defines data elements required to align your business objectives with Amazon Fraud Detector's data models.",
+        description: 'step 2 description',
+      }),
+      buttonTitle: intl.formatMessage({
+        defaultMessage: 'Explore widgets',
+        description: 'Explore widgets button title',
+      }),
+      handleCallback: () => {
+        //TODO
+      },
+    },
+    {
+      columnTitle: intl.formatMessage({
+        defaultMessage: 'Step 3: Browse and add your assets',
+        description: 'step 3 title',
+      }),
+      icon: AssetsSearch,
+      className: 'getting-started-step3-icon',
+      extraMargin: '14px',
```

Comment:
use design token ?

Reply:
removed extraMargin since we are already taking it from  styles.css. file